content/search-fitness-activities.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Search Fitness Activities
+---
+
+* TOC
+{:toc}
+
+## Retreive a list of Fitness Activities
+
+    GET https://everydayhero.com/api/v2/search/fitness_activities
+
+### Parameters
+
+charity_id : _optional_ **string** or **Array**<br/>
+Charity ID, only totals for that the specified Charity will be retrieved.
+An Array of IDs can be provided, eg: charity_id[]=au-1&charity_id[]=au-2
+
+campaign_id : _optional_ **string** or **Array**<br/>
+Campaign ID, only totals for that the specified Campaign will be retrieved.
+An Array of IDs can be provided, eg: campaign_id[]=au-1&campaign_id[]=au-2
+
+team_id : _optional_ **string**<br/>
+team_uid : _optional_ **string**<br/>
+team_page_id : _optional_ **string**<br/>
+Team ID, only totals for that the specified Team will be retrieved.
+
+start_at : _optional_ **string/date-time**<br/>
+Start at date, ISO format: "2014-08-20 05:02:13", filter totals by a range.
+
+end_at : _optional_ **string/date-time**<br/>
+End at date, ISO format: "2014-08-20 05:02:13", filter totals by a range.
+
+type : _optional_ **string**<br/>
+Filter by activity type: e.g. bike, gym, hike, run, sport, swim, walk
+
+include_manual : _optional_ **boolean**<br/>
+Include manually entered activities. Default: false.
+
+exclude_virtual : _optional_ **boolean**<br/>
+Exclude virtual activities. Default: false
+
+exclude_non_virtual : _optional_ **boolean**<br/>
+Exclude non-virtual activities. Default: false
+
+#### Pagination
+
+For more information about pagination, please see the [pagination
+overview](/overview/#pagination).
+
+limit : _optional_ **integer**<br/>
+The number of items per page.
+
+page : _optional_ **integer**<br/>
+The page number to return.
+
+### Example
+
+    https://everydayhero.com/api/v2/search/fitness_activities.json?campaign_id=ie-0
+
+### Response
+
+<%= json :search_fitness_activities %>
+
+## Retreive a Fitness Activities Leaderboard
+
+    GET https://everydayhero.com/api/v2/search/fitness_activities_totals
+
+### Parameters
+
+Includes all above paramaters as well as
+
+group_by : _optional_ **string**<br/>
+Group results by pages or teams. Default: pages
+
+sort_by : _optional_ **string**<br/>
+Sort results by distance_in_meters, duration_in_seconds, elevation_in_meters or
+calories. Default: distance_in_meters
+
+
+### Example
+
+    https://everydayhero.com/api/v2/search/fitness_activities_totals.json?campaign_id=ie-0&type=bike&group_by=page
```

Comment:
{
"results": []
}

:[
might be nice to use us or something here that has results now, not a big deal though